### PR TITLE
Update playwright/testcafe/puppeteer version for docker mode

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |Playwright Version|Supported Browsers|
 |-----|----|
+|1.11.1|Please see [release notes](https://github.com/saucelabs/sauce-playwright-runner/releases/tag/v1.12.0)|
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit.md:149]: Another double-check on this....first column is 1.11.1, but the link tag is 1.12.0. Is that correct?

File: docs/testrunner-toolkit.md [hunk 2]
```diff
@@ -156,6 +157,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |Puppeteer Version|Supported Browsers|
 |-----|----|
+|9.1.1|Please see [release notes](https://github.com/saucelabs/sauce-puppeteer-runner/releases/tag/v1.1.0)|
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit.md:160]: Same check on the runner version vs. the fw version. :)

**Review by sweeneyskirt-sl — COMMENTED** [review]
Based on Felix's explanation of the runner version and fw version not necessarily being in sync, I am sure these are all correct. Just wanted to double check. ;)

File: docs/testrunner-toolkit.md (hunk 1 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit.md:149]: correct. It's `1.11.1` for playwright framework version, and `1.12.0` for playwright runner version

File: docs/testrunner-toolkit.md (hunk 2 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit.md:160]: They're not matching each other. so it's correct :)

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**